app/src/main/java/org/simple/clinic/login/pin/LoginPinScreenController.kt

```diff
@@ -45,7 +46,7 @@ class LoginPinScreenController @Inject constructor(
 
   private fun screenSetups(events: Observable<UiEvent>): Observable<UiChange> {
     return events.ofType<PinScreenCreated>()
-        .flatMapSingle {
+        .flatMapSingle { _ ->
```

Comment:
Did you intentionally add this?

Reply:
here and in many other places

Reply:
Yes. I got a lot of warnings from the Kotlin 1.2.61 plugin, which I updated today.